about.html

```diff
@@ -27,39 +26,3 @@
         <a href="sign-in.html">Sign-in</a>
       </nav>
     </header>
-=======
-    <body>
-      <div class="top flex">
-          <div class="container flex">
-              <div class="grow"></div>
-
-      <img src="images/KidsFlyLogo1.JPG" title="logo" />
```

Comment:
Hey Tammy

if you are delete this, "about html" does not have same header like home page had